lib/prototype-rails/on_load_action_view.rb

```diff
@@ -14,9 +14,11 @@
   include ActionView::Helpers::ScriptaculousHelper
 end
 
-ActionView::TestCase.class_eval do
-  include ActionView::Helpers::PrototypeHelper
-  include ActionView::Helpers::ScriptaculousHelper
+if defined?(Rails) && Rails.env.test?
```

Comment:
the idea was to only load it in test mode, so `defined?(Rails.env)` will also be true for development and others

how about `['cucumber', 'test'].include?(Rails.env)` ?
there is always the fallback option of doing the include manually, so even if we miss some edge case it's not that bad